modules/wechat_qrcode/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/src)
```

Comment:
Not allowed. Use relative paths instead.

See mess with precomp headers:

```
In file included from /Volumes/build-storage/build/precommit-contrib_ios/opencv/modules/gapi/src/api/gframe.cpp:8:
In file included from /Volumes/build-storage/build/precommit-contrib_ios/opencv_contrib/modules/wechat_qrcode/src/precomp.hpp:25:
```

Reply:
Thanks a lot! I was really confused yesterday about the compile failure.

Reply:
Relative paths should be used in .cpp/.hpp files for `#include` directives.

Reply:
I know your meaning. I tried to save time by modifying CMakeLists instead of `#include` (As you can see, there are many files under the zxing module). However, I failed. Now, I have changed them with `#include` relative paths. Thanks!